### PR TITLE
chore: bump candy machine deps

### PR DESCRIPTION
**Pull request by samuelvanderwaal**

### CODE REVIEW

**Review by samuelvanderwaal — COMMENTED** [review]
version minor

File: candy-machine/js/idl/candy_machine.json [hunk 1]
```diff
@@ -363,7 +363,7 @@
         },
         {
           "name": "collectionMetadata",
-          "isMut": true,
+          "isMut": false,
```
Comment (stegaBOB) [inline candy-machine/js/idl/candy_machine.json:366]: this is not good...

Comment (stegaBOB) [reply candy-machine/js/idl/candy_machine.json:366]: how did it get flipped?

Comment (samuelvanderwaal) [reply candy-machine/js/idl/candy_machine.json:366]: Did the Solita hook not run?

**Review by stegaBOB — APPROVED** [review]

**Review by blockiosaurus — APPROVED** [review]

**Pull request merged**